pages/Home.vue

```diff
@@ -198,6 +211,17 @@ export default {
   },
   mounted () {
     if (!this.isLoggedIn && localStorage.getItem('redirect')) { this.$bus.$emit('modal-show', 'modal-signup'); }
+    this.banners = this.promotedOffers.mainBanners
+    this.heroes = this.heroImages
+    supportsWebP.then(supported => {
```

Comment:
Can you do it inside some module, for example `'ui'`?
This can be done just once when module is initialized and stored in vuex. 
`await store.dispatch('ui/checkWebpSupport')`
and then we can get from vuex by getter `isWebpSupported`.

This lib probably tries to create data:image/webp and fetch it. It takes some miliseconds, but if this will be done in more components it can make a lot of small requests. If we will have `isWebpSupported` in vuex then we can also use it in future for products :)

Reply:
You are right, I moved this to store module